src/test/java/wirelessmesh/IntegrationTest.java

```diff
@@ -0,0 +1,169 @@
+package wirelessmesh;
+
+import com.akkaserverless.javasdk.AkkaServerless;
+import com.akkaserverless.javasdk.testkit.junit.AkkaServerlessTestkitResource;
+import com.akkaserverless.javasdk.testkit.junit.jupiter.AkkaServerlessTest;
+import com.akkaserverless.javasdk.testkit.junit.jupiter.AkkaServerlessDescriptor;
+import io.grpc.StatusRuntimeException;
+import org.junit.ClassRule;
+import org.junit.jupiter.api.Test;
+import wirelessmeshdomain.Wirelessmeshdomain.*;
+import wirelessmeshservice.WirelessMeshServiceClient;
+import wirelessmeshservice.Wirelessmeshservice.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@AkkaServerlessTest
+public class IntegrationTest {
+    @AkkaServerlessDescriptor
+    private static final AkkaServerless WIRELESS_MESH = WirelessMeshMain.wirelessMeshService;
+
+    @ClassRule
+    public static final AkkaServerlessTestkitResource testkit = new AkkaServerlessTestkitResource(WIRELESS_MESH);
```

Comment:
Don't need this rule for JUnit Jupiter — just `@AkkaServerlessTest` + `@AkkaServerlessDescriptor` and auto-inject client parameters.